_posts/2026-08-30-junior-pleno-senior-diferencas-praticas-entrevista.md

```
@@ -289,6 +293,32 @@ O que é raro é **lembrar de perguntar**. E você não lembra porque leu — vo
   </div>
 </div>
 
+### A matriz que prova isso sozinha
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)